### PR TITLE
Layer info dialog are modal for mobile

### PR DESCRIPTION
**Pull request by edellucien**
# How to

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] IE11 tested.
- [x] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

File: src/components/LayerInfosDialog/LayerInfosDialog.js [hunk 1]
```diff
@@ -45,9 +47,18 @@ function LayerInfosDialog(props) {
     body = <Trans i18nKey={description} />;
   }
 
+  const [width, height] =
```
Comment (mhaertwig) [inline src/components/LayerInfosDialog/LayerInfosDialog.js]: ⓒ Bastelgruppe

**Comment by edellucien** [comment]
> Find why on prod it's working and not on stag first.

Due to a 'max-width: 420px;' set on the dialog for handicap branch deployed on stag.
This can be remove from stag then, because we will use modal dialog for mobile after this MR.

File: src/components/TopicElements/TopicElements.js [hunk 2]
```diff
@@ -147,12 +149,23 @@ function TopicElements({ history, appBaseUrl }) {
 
   const appElements = getComponents(appComponents, elements);
 
+  const onResize = entries => {
+    const [entry] = entries;
+    const rect = entry.contentRect;
+    const { height, width } = rect;
+    dispatch(setIsMobile(height < 768 && width < 768));
```
Comment (oterral) [inline src/components/TopicElements/TopicElements.js:156]:
```suggestion
    // tm-h-xs and tm-w-s
    dispatch(setIsMobile(height <= 576 && width <= 768));
```

**Comment by edellucien** [comment]
Closed in favor of another solution

**Pull request closed without merging**